decidim-elections/app/forms/decidim/elections/admin/censuses/internal_users_form.rb

```diff
@@ -66,11 +66,14 @@ def options_manifest(handler_name)
           def authorization_is_valid
             return if authorization_handlers_names.blank?
 
-            valid_types = context.current_organization.available_authorizations
-            invalid_types = (authorization_handlers_names - valid_types).compact_blank
+            invalid_types = (authorization_handlers_names - organization_authorizations).compact_blank
 
             errors.add(:base, :invalid) if invalid_types.present?
           end
+
+          def organization_authorizations
```

Comment:
I think this could be a one liner, as we already `delegate :current_organization, to: :context` in the parent class: 
```
  delegate :available_authorizations, to: :current_organization, prefix: :organization
```

Then use it where needed as `organization_available_authorizations`.

Reply:
ok